packages/web3-validator/package.json

```diff
@@ -21,7 +21,8 @@
 	},
 	"files": [
 		"lib/**/*",
-		"src/**/*"
+		"src/**/*",
+		"dist/**/*"
```

Comment:
After a PR in past where lib and dist directors both are created for outputs, dist was not whitelisted, so due to current CDN structure web3.min is not available in last few RCs as its not uploaded to npm due to filtering, 

Its fixed here. 

tested `dist/min..` including via` yarn pack`